### ISSUE TITLE
Add favourited finishers into loadouts

### ISSUE THREAD

**Issue by DragonXDoom** [DestinyItemManager/DIM#8110]
### Proposed change

Hey there,

DIM already has the ability to change which finishers are favourited or not.
I would love to, as well, have the ability to add finishers as part of a loadout.

I am not quite sure how this would work with collections (I'm unsure if DIM can pull finishers from collections or not) or if DIM would strictly be working with what finishers you have in your "slots".

### How does this fit into your workflow?

I have certain finishers I like to use only on certain loadouts. For example, I love the Golden Gun finisher, but I prefer only to use it when I'm actually on a solar subclass. If I'm on a more melee based build (for example, arcstrider with Liar's Handshake) I like my finishers to include lots of punches and kicks.

The short version is: I want to be able to customise which finishers are used on each loadout, without having to manually adjust them in game.

**Comment by bhollis** [comment]
Duplicate of #7436
